cdc_rsync_server/main.cc

```diff
@@ -18,6 +18,16 @@
 #include "common/log.h"
 #include "common/status.h"
 
+#define DEV_BUILD_VERSION "DEV"
+#ifdef CDC_BUILD_VERSION
+#define TO_STR(arg) #arg
+#define TO_STR_VALUE(arg) TO_STR(arg)
```

Comment:
Since this will be used in both client and server (and later in cdc_stream, I suppose), I would suggest to move it to a separate .h file in common.